app/Services/Manager.php

```diff
@@ -495,4 +495,41 @@ public function addUserToModel($model, $id, $user)
 
         return true;
     }
+
+    /**
+     * Fires off an event to send an email a user.
+     *
+     * @param  Gladiator\Models\User     $user
+     * @param  Gladiator\Models\Contest  $contest
+     * @param  array $params
+     * @return null|void
+     */
+    public function sendEmail($user, $contest, $params)
+    {
+        if (! isset($params)) {
+            Log::error('Gladiator\Services\Manager -- No params passed');
```

Comment:
We could eventually throw an exception here instead of just logging. One that is caught by the Handler.php but doesn't break out of the processing! But not necessary for merge. Just noting :)